### ISSUE TITLE
anyone know how to istall this drive on openwrt?

### ISSUE THREAD

**Issue by jinglei207** [brektrou/rtl8821CU#28]

**Comment by josenicomaia** [comment]
+1

**Comment by yu13740000** [comment]
+1

**Comment by AddaxSoft** [comment]
+1

**Comment by tony1016** [comment]
+1

**Comment by dausruddin** [comment]
Try this perhaps?

https://github.com/immortalwrt/immortalwrt/tree/d799ed3ce59c85929facc30a8834f3838c7a95a7/package/ctcgfw/rtl8821cu

**Comment by AddaxSoft** [comment]
please report back if it worked!

**Comment by dausruddin** [comment]
I actually compiled it into my firmware succesfully, but my usd adapter wouldn't work. Would be great if someone can tell me what packages are essentials to have wifi working on openwrt system, because I might be missing some packages.

**Comment by tony1016** [comment]
lsusb -t show driver status

dausruddin <notifications@github.com> 于 2021年3月3日周三 下午10:50写道：

**Comment by dausruddin** [comment]
I had my friend tested the firmware. Today I want to rebuild my firmware with the driver mentioned above and it failed to build.

**Comment by tony1016** [comment]
> please report back if it worked!

yes,it works on 19.07.7

**Comment by tony1016** [comment]
> I had my friend tested the firmware. Today I want to rebuild my firmware with the driver mentioned above and it failed to build.
> I got the same error as this [immortalwrt/immortalwrt#249](https://github.com/immortalwrt/immortalwrt/issues/249)
> 
> Must be something from openwrt repo. At this point, I am thinking to quit as this is my limit of knowledge. My RTL8811CU USB adapter is on its way to my home though XD

I build it with offical 19.07.0 successfully.And it's perfect,THX

**Comment by tony1016** [comment]
My build 
https://drive.google.com/file/d/1pV5FvWZODYOcsZuQuvrT1WZPIAwwvEjW/view?usp=sharing

**Comment by dausruddin** [comment]
Guys! If you are not able to get wifi working, see this issue https://github.com/immortalwrt/immortalwrt/issues/299.

TL;DR:
- Check kernel log after plugging in the adapter.
- If the adapter [detected as storage](https://github.com/immortalwrt/immortalwrt/issues/299#issuecomment-791137162), install [usb-modeswitch](https://openwrt.org/packages/pkgdata/usb-modeswitch).

It is working for me now. You might also need `wpad` and `iwinfo` as mentioned in link of the issue I attached above.
Give feedback on your progress!

**Comment by Boos4721** [comment]
dude, anyone knows about rtl88x2bu on openwrt ?

**Comment by trongbang2** [comment]
> My build https://drive.google.com/file/d/1pV5FvWZODYOcsZuQuvrT1WZPIAwwvEjW/view?usp=sharing

Sorry, how to install this driver in openwrt 19.07.4
I got the Executing package manager error

Unknown package 'kmod-rtl8821cu'.

Errors

Collected errors:
 * pkg_hash_fetch_best_installation_candidate: Packages for kmod-rtl8821cu found, but incompatible with the architectures configured
 * opkg_install_cmd: Cannot install package kmod-rtl8821cu.

The opkg install command failed with code 255

Thank you!.
